Fix:

meson.build:46:10: ERROR: pkgconfig.generate got unknown keyword arguments "install"

Old meson versions ignored this invalid parameter; new versions error.

https://gitlab.gnome.org/GNOME/libhttpseverywhere/-/issues/3
--- meson.build.orig	2018-04-13 15:53:47.000000000 -0500
+++ meson.build	2025-08-20 17:19:59.000000000 -0500
@@ -49,8 +49,7 @@
                    filebase : meson.current_build_dir()+'/httpseverywhere-'+api,
                    requires : 'glib-2.0 gobject-2.0 gio-2.0 json-glib-1.0 libsoup-2.4 gee-0.8 libarchive',
                    subdirs: 'httpseverywhere-'+api,
-                   description : 'A library to rewrite HTTP URLs to HTTPS URLs.',
-                   install: true)
+                   description : 'A library to rewrite HTTP URLs to HTTPS URLs.')
 
 install_data('data/default.rulesets', install_dir: get_option('datadir') + '/libhttpseverywhere')
 install_data('httpseverywhere-'+api+'.deps', install_dir: get_option('datadir') + '/vala/vapi')
